sdk/changeanalysis/Microsoft.Azure.Management.ChangeAnalysis/src/Microsoft.Azure.Management.ChangeAnalysis.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+	<!-- Please do not move/edit code below this line -->
+	
+	<PropertyGroup>
+		<Description>Provides management capabilities for Azure Change Analysis</Description>
+		<AssemblyTitle>Microsoft Azure Change Analysis Management Library</AssemblyTitle>
+		<Version>1.0.0</Version>
```

Comment:
The first version should be 1.0.0-beta.1

Reply:
> Please add your project into the management pipeline through this file: https://github.com/Azure/azure-sdk-for-net/blob/master/eng/pipelines/mgmt.yml#L41

I added it in another [PR](https://github.com/Azure/azure-sdk-for-net/pull/19660/files) since the bot said
> Please open a separate PR and to your service SDK path in this file. Once that PR has been merged, you can re-run the pipeline to trigger the verification.

I'll close that PR if we can simply add that line in this PR.

I updated the version to 1.0.0-beta.1, just to double check, we can leave the AssemblyVersion as 1.0.0.0 in AssemblyInfo.cs correct?